File: frontend/app/src/app/features/password-entry/interfaces/passwordEntryDto.ts
import { IEntity } from '../../../core/interfaces/entity';

export interface IPasswordEntryDto extends IEntity {
  entryName?: string;
  websiteUrl?: string;
  entryUserName?: string;
  encryptedPassword?: Uint8Array;
  iv?: Uint8Array;
  notes?: string;
}

File: backend/API/Dtos/PasswordEntry/PasswordEntryDto.cs
namespace API.Dtos.PasswordEntry;

public class PasswordEntryDTO : ModelDTO
{

    public string? EntryName { get; set; }
    public string? WebsiteUrl { get; set; }
    public string? EntryUserName { get; set; }
    public byte[]? EncryptedPassword { get; set; }
    public string? IV { get; set; }
    public string? Notes { get; set; }
}